tests/end2end/helpers/E2ETopics.js

```diff
@@ -44,14 +52,19 @@ export class E2ETopics {
         E2EApp.confirmationDialogAnswer(confirmDialog);
     }
 
+    static label2TopicEnterFreetext(labelName) {
+        browser.element("#id_item_selLabels").click();
+        browser.keys(labelName+"\uE007"); // plus ENTER
+    }
+
     static responsible2ItemEnterFreetext(theText) {
         E2EGlobal.waitSomeTime();
 
         // &%$#$@! - the following does not work => Uncaught Error: element not visible
         // browser.element(".form-group-responsibles .select2-selection").click();
         // ... so we take this as workaround: click into first select2 then Tab/Tab to the next one
 
-        browser.element(".form-group-labels .select2-selection").click();
+        browser.element("#id_selResponsible").click();
```

Comment:
Revert this change and remove the class `form-group-labels` from the topicEdit-Template in line 17. This should make the tests green again.

Alternatively you could remove this dirty workaround and use the same nice trick as you did for writing text into the topic-labels field: clicking on the label of this select element. I would prefer this solution! (Remember giving the label its own id)